src/165490.c

```diff
@@ -681,15 +682,15 @@ void filemenu_main_handle_input(MenuPanel* menu) {
                     sfx_play_sound(SOUND_MENU_NEXT);
                     set_game_mode(0xD);
                 } else if (menu->selected < 4) {
-                    if (D_80077A24[menu->selected] == 0) {
+                    if (gSaveSlotHasData[menu->selected] == 0) {
                         sfx_play_sound(SOUND_MENU_NEXT);
                         filemenu_8024C098 = 2;
                         filemenu_menus[filemenu_8024C098]->page = 1;
                         set_window_update(0x2C, main_menu_window_update);
                         set_window_update(0x2F, 1);
                         set_window_update(0x32, 2);
                         fio_save_game(menu->selected);
-                        D_80077A24[menu->selected] = 1;
+                        gSaveSlotHasData[menu->selected] = 1;
```

Comment:
if this is a bool, all of the checks can be ! for == 0 and the setting can be TRUE/FALSE instead of 1/0

Reply:
good suggestion, but that code is in a NON_EQUIVALENT block and im reluctant to interefere with it until someone can match it